glean-core/pings.yaml

```diff
@@ -39,8 +39,24 @@ metrics:
     - https://bugzilla.mozilla.org/1512938
   data_reviews:
     - https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3
+    - https://bugzilla.mozilla.org/show_bug.cgi?id=1557048#c13
   notification_emails:
     - glean-team@mozilla.com
+  reasons:
+    overdue: |
+      The ping wasn't submitted on the current calendar day, but it's after 4am,
+      so it's submitted immediately
+    today: |
+      The ping wasn't submitted on the current calendar day, but it is still
+      before 4am, so schedule to send on the current calendar day at 4am.
```

Comment:
The tenses of these are a little weird. Shouldn't they be describing the ping they're on, not the scheduling scenario? "The ping wasn't submitted..." but then how did we get it? The ping must have been submitted.

Maybe reword a la: "The ping was submitted after having been scheduled earlier than 4am today." or something that makes it about the value that's in this ping.

Reply:
Good point.  I've rephrased this in terms of "last ping" and "this ping", which hopefully makes things clearer.